### PR TITLE
ru translate pages:

### PR DESCRIPTION
**Pull request by Smekalin**
- getting started
- examples
- videos
- single-spa playground
- part of sidebar

### CODE REVIEW

**Comment by Smekalin** [comment]
@joeldenning I can't select @astra1 as a reviewer :\

**Comment by joeldenning** [comment]
@astra1 hasn't accepted their invitation to this repository yet, which is why you can't assign them as a reviewer.

File: website/versioned_docs/version-5.x/examples.md [hunk 1]
```diff
@@ -1,37 +1,37 @@
 ---
 id: examples
-title: Single-spa Examples
-sidebar_label: Resources
+title: Примеры использования single-spa
+sidebar_label: Ресурсы
 ---
 
-## Core team examples
+## Примеры от основной команды
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:7]: @Smekalin maybe replace `основной` with `ядра`?

File: website/versioned_docs/version-5.x/examples.md [hunk 2]
```diff
@@ -1,37 +1,37 @@
...
+- [Isomorphic Microfrontends](https://github.com/isomorphic-microfrontends) пример рендеринга на стороне сервера (SSR) с single-spa и single-spa-layout.
 
-### Older examples
+### Старые примеры
 
-- [coexisting-angular-microfrontends](https://github.com/joeldenning/coexisting-angular-microfrontends) is a full blown Angular 9 microfrontends repo that combines three separate Angular CLI projects into one page.
-- [coexisting-vue-microfrontends](https://github.com/joeldenning/coexisting-vue-microfrontends) shows three separate Vue CLI projects existing within one page.
-- [single-spa-portal-example](https://gitlab.com/TheMcMurder/single-spa-portal-example) is a great example of coexisting React microfrontends.
-- [simple-single-spa-webpack-example](https://github.com/joeldenning/simple-single-spa-webpack-example) is a small, simple example that can be used as a webpack starter.
+- [coexisting-angular-microfrontends](https://github.com/joeldenning/coexisting-angular-microfrontends) это полноценный репозиторий микрофронтенда с Angular 9 который совмещает три отдельных Angular CLI проекта на одной странице.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:19]: @Smekalin please, consider to replace `отдельных` with `различных`

File: website/versioned_docs/version-5.x/examples.md [hunk 3]
```diff
@@ -1,37 +1,37 @@
...
+- [coexisting-vue-microfrontends](https://github.com/joeldenning/coexisting-vue-microfrontends) Показывает три отдельных Vue CLI проекта существующих на одной странице.
+- [single-spa-portal-example](https://gitlab.com/TheMcMurder/single-spa-portal-example) это отличный пример сосуществования трех микрофронтендов на React.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:21]: @Smekalin consider to replace end with `трех React микрофронтендов`

File: website/versioned_docs/version-5.x/examples.md [hunk 4]
```diff
@@ -1,37 +1,37 @@
...
-- [single-spa application with shared styled-components](https://github.com/filoxo/single-spa-example-shared-styled-components) shows how to share styled-components, a library that is required to be a singleton instance.
-- [single-spa application with Webpack lazyStyleTag](https://github.com/filoxo/single-spa-example-webpack-lazystyletag) is a simple example that leverages Webpack style-loader's lazyStyleTag functionality to dynamically add and remove the CSS associated with a single-spa application.
-- [single-spa shared state utility using Rxjs](https://github.com/filoxo/single-spa-example-rxjs-shared-state) shows how to use a utility module that shares application state across multiple single-spa applications and frameworks.
+- [single-spa-parcel-example](https://github.com/Guillembonet/single-spa-parcel-example) является примером микрофронтендов на Vue и React, содержащие Vue и React parcels соответственно и два Node.js микросервиса, запущенных на 6 разных Docker виртуальных машинах и легко работающих вместе в одном веб-приложении, размещенном на 7-ой виртуальной машине.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:26]: @Smekalin we must rearrange the sentence and use some common naming for `parcel` like `участок`, `зона` or better variant. Please, look at [multitran](https://www.multitran.com/m.exe?l1=1&l2=2&s=parcel)

File: website/versioned_docs/version-5.x/examples.md [hunk 5]
```diff
@@ -1,37 +1,37 @@
...
+- [single-spa-login-example-with-npm-packages](https://github.com/jualoppaz/single-spa-login-example-with-npm-packages) это пример single-spa приложения, которое импортирует зарегистрированные приложения из пакетов NPM и управляет функциями аутентификации в качестве входа в систему.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:27]: @Smekalin consider to replace `зарегистрированные ` with `указанные`

File: website/versioned_docs/version-5.x/examples.md [hunk 6]
```diff
@@ -1,37 +1,37 @@
...
+- [demo-single-spa-with-spax](https://github.com/crossjs/spax/tree/master/packages/demo-single-spa) это маленький [spax](https://spax.js.org) пример с react-scripts и craco.
+- [single-spa-html with js example](https://github.com/filoxo/single-spa-html-with-js-example) это пример репозитория использует single-spa-html, который дополнен простым JavaScript.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:29]: @Smekalin consider to replace `простым` with `чистым`

File: website/versioned_docs/version-5.x/examples.md [hunk 7]
```diff
@@ -1,37 +1,37 @@
...
+- [coexisting-angular-microfrontends/login](https://github.com/Vallerious/coexisting-angular-microfrontends/tree/feature/login) эта ветка реализует функционал авторизации между приложениями на Angular. Используется localStorage как общее пространство памяти, чтобы сохранять и получать токен.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:30]: @Smekalin you've used `аутентификации` before, `авторизация` is different process. And replace a second sentence with `LocalStorage API используется как общее пространство памяти для получения и сохранения токена.`

File: website/versioned_docs/version-5.x/examples.md [hunk 8]
```diff
@@ -1,37 +1,37 @@
...
+- [single-spa-angular-cli](https://github.com/matt-gold/single-spa-angular-cli) это пример полностью Angular репозитория, который использует SystemJs, чтобы загрузить single-spa-angular приложения в содержащее Angular CLI приложение по различным роутам.
+- [ember-micro-frontends](https://github.com/ember-micro-frontends) это пример полностью Ember репозитория, который использует single-spa-ember приложения в содержащее Ember.js приложение по различным роутам.
+- [single-spa application with shared styled-components](https://github.com/filoxo/single-spa-example-shared-styled-components) показывает как совместно использовать styled-components, библиотека, которая обязательно должна быть синглтоном.
+- [single-spa application with Webpack lazyStyleTag](https://github.com/filoxo/single-spa-example-webpack-lazystyletag) это простой пример, который использует функциональность lazyStyleTag Webpack style-loader для динамического добавления и удаления CSS, связанного с одним single-spa приложением.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:34]: @Smekalin please, consider to replace `...lazyStyleTag Webpack style-loader` with `...загрузчика стилей вебпака lazyStyleTag...`

File: website/versioned_docs/version-5.x/examples.md [hunk 9]
```diff
@@ -1,37 +1,37 @@
...
+- [single-spa shared state utility using Rxjs](https://github.com/filoxo/single-spa-example-rxjs-shared-state) показывает, как использовать служебный модуль, который совместно использует состояние приложения в нескольких однокомпонентных приложениях и фреймворках.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:35]: @Smekalin `использовать как модуль, который совместно использует` looks unnatural for my ears :)

File: website/versioned_docs/version-5.x/examples.md [hunk 10]
```diff
@@ -1,37 +1,37 @@
...
 
-Have your own example or starter repo? [Submit a PR](https://github.com/single-spa/single-spa.js.org/edit/master/website/versioned_docs/version-5.x/examples.md) to add yours to this list.
+У вас есть свой пример или стартер-репозиторий? [Отправить PR](https://github.com/single-spa/single-spa.js.org/edit/master/website/versioned_docs/version-5.x/examples.md), чтобы добавить ваш пример в данный список.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/examples.md:37]: @Smekalin consider to replace `стартер-репозиторий` with `репозиторий для старта`

File: website/versioned_docs/version-5.x/single-spa-playground.md [hunk 11]
```diff
@@ -6,8 +6,8 @@ sidebar_label: Playground
...
 # Single-spa Playground
 
-[Playground website](http://single-spa-playground.org)
+[Playground сайт](http://single-spa-playground.org)
 
-[Github project](https://github.com/single-spa/single-spa-playground)
+[Github проект](https://github.com/single-spa/single-spa-playground)
 
-The single-spa playground is an interactive tutorial that lets you test your code live in the browser to verify it can work with single-spa.
+Single-spa playground это интерактивное руководство, которое позволяет вам тестировать ваш код браузере в реальном времени, чтобы проверить, как он может работать с single-spa.
```
Comment (astra1) [inline website/versioned_docs/version-5.x/single-spa-playground.md:13]: @Smekalin consider to replace `чтобы проверить, как он может работать с single-spa` with `чтобы проверить интеграцию с single-spa`

File: website/versioned_docs/version-5.x/videos.md [hunk 12]
```diff
@@ -1,19 +1,19 @@
...
 ---
 
-# Video Tutorials
+# Видео-руководства
 
-A variety of video tutorials exist from both the single-spa core team and other community members.
+Существует множество видеоуроков как от основной команды single-spa, так и от других членов сообщества.
 
-## From the Core Team
+## От основной команды
```
Comment (astra1) [inline website/versioned_docs/version-5.x/videos.md:11]: @Smekalin same `ядра` usage

**Review by astra1 — CHANGES_REQUESTED** [review]
@Smekalin @joeldenning review is done for now, thank you in advance!

**Comment by Smekalin** [comment]
@astra1 Thanks for the quick respond! I've done.

**Review by astra1 — APPROVED** [review]
Legitimate 💓

**Comment by joeldenning** [comment]
Feel free to merge these pull requests yourselves - you don't need to wait for me to merge it.

**Pull request merged**